src/can_common.h

```diff
@@ -1,13 +1,36 @@
 #ifndef _CAN_COMMON_H_
 #define _CAN_COMMON_H_
 
-#define CAN_THROTTLE        45
-#define CAN_MOTOR_ENABLE    20
-#define CAN_ACCESSORIES
-#define CAN_TELEMETRY       0xA0
-
-#define CAN_BMS_DEFAULT_ID  0x01
-#define CAN_BMS_REQUEST(x)  (0x200 + (x))
-#define CAN_BMS_RETURN(x)   (0x240 + (x))
+// Motor Enable 
+// 0x0 - 0x1F
+// 0x100 - 0x11F
+// NO MESSAGES DEFINED
+
+// Throttle
+// 0x20 - 0x3F
+// 0x120 - 0x13F
+// NO MESSAGES DEFINED
+
+// Motor Controller
+// 0x40 - 0x5F
+// 0x140 - 0x15F
+// NO MESSAGES DEFINED
+
+// Accessories
+// 0x60 - 0x7F
+// 0x160 - 0x17F
+#define CAN_ACC_OPERATION       0x60
+#define CAN_ACC_STATUS          0x71
```

Comment:
Did we want to maintain a split between high and low priority messages for the same ID? If so, we should add a macro that sets the 9th bit high.

Reply:
We're still maintaining that split in the ID assignments, but it's up to each team how they want to use them. We have very few IDs actually being used, so I don't think priority conflicts should be an issue. These definitions are based on what Dora and Roy decided for the CAN Accessories boards, for example.